static/styles/left_sidebar.css

```diff
@@ -639,7 +677,34 @@ li.expanded_private_message {
         display: none;
     }
 
+    #more_private_messages_sidebar_title {
+        display: inline;
+    }
+
+    .hide_more_private_messages {
+        display: block;
+        text-decoration: none;
+        color: inherit;
+        margin-left: 13px;
+        opacity: 0.7;
+
+        .fa-chevron-left {
+            padding-right: 3px;
+        }
+    }
+
     .zoom-in-hide {
         display: none;
     }
+
+    .zoom-in-sticky {
+        position: sticky;
+        top: 0;
+        z-index: 1;
+        padding: 3px 0 3px $far_left_gutter_size;
+    }
+
+    #show_all_private_messages {
+        margin-right: 12px !important;
```

Comment:
Why does this need `!important`? Usually that's not necessary for a CSS rule on an ID.

Reply:
I tried avoiding this but wasn't able to, actually we need to shift this icon a bit in zoomedIn view but it already has got some margin-right with it in zoomedOut view.

Reply:
I'll try to have a look at it once again if I can avoid it somehow.